src/styles/global.scss

```diff
@@ -0,0 +1,26 @@
+body {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: -apple-system, BlinkMacSystemFont,'DM Sans', sans-serif, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans",
+    "Droid Sans", "Helvetica Neue", sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  background-color: #F7F8FA;
+}
+
+code {
+  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New", monospace;
+}
+
+h2{
+  font-size: 30px;
+  font-weight: 400;
+  margin-top: 5px;
+}
+
+p{
+  font-size: 16px;
+  font-weight: 400;
+  margin-top: 5px;
+}
```

Comment:
I think some global rules are ok. I do not like the globals margins/padding though. Mainly because each h2, p and such will get specific styling depending on the screen, so you may end up having to add styles just to reverse this one.

Also, SCSS files are not linted by `standard`, because they are not JS. So you must enforce some standard rules yourself. For example, a space before `{` is required, both for `h2` and `p`.